bower.json

```diff
@@ -34,7 +34,7 @@
     "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.3.3",
     "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.2.0",
     "vaadin-license-checker": "^2.1.0",
-    "pdfjs-dist": "2.0.943",
+    "pdfjs-dist": "2.1.266",
```

Comment:
We should also submit a PR to `bundled-es-modules`:
https://github.com/bundled-es-modules/pdfjs-dist

An then update it the [dependency-map.json](https://github.com/vaadin/magi-cli/blob/30942998bf2f27173469a5d0e427f0a931433a18/dependency-map.json#L16) in magi.